### PR TITLE
Creating a doc template for community members to use

### PR DESCRIPTION
**Pull request by asachs01**

### CODE REVIEW

**Comment by asachs01** [comment]
Fixes #177

File: static/files/sensu-doc-template.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---    
+# Header 1
+When you write a doc and link to a site, it should look like this: `[sensuapp.org][1]`, so functionally: [sensuapp.org][1]
+
+# Header 2
+If you create code that needs highlighting, it should look like this:
+
+```
+ {{< highlight shell >}}
+ /usr/bin/whoami{{< /highlight >}}
```
Comment (mbbroberg) [inline static/files/sensu-doc-template.md]: It'd be helpful to mention any gotchas with highlighting (on the same line or can I new-line it?)

File: static/files/sensu-doc-template.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+Optionally, you can change this to ` {{< highlight json >}}` for highlighting json text.
```
Comment (mbbroberg) [inline static/files/sensu-doc-template.md]: This took too long for me to understand. It'd be great mention the use of `/hightlight` vs `highlight $SPECIFIC`.

File: static/files/sensu-doc-template.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+# Header 3
+If you are documenting an API endpoint, use the following format:
```
Comment (mbbroberg) [inline static/files/sensu-doc-template.md]: This is super overwhelming. Maybe some text beforehand mentioning that "You'll rarely want or need to do this, but here's a very complex example of syntax highlighting that we've standardized across the project.

**Pull request merged**